### PR TITLE
[Frost Mage] Cryopathy talent

### PR DESCRIPTION
**Pull request by earosselot**
### Description

Added Cryopathy talent statistics.
Minor fixes:
- Splitting ice icon
- Updated example report

### Testing

- Test report URL: `/report/RJ1MXnb2zymgQ8pF/49-Mythic+Fyrakk+the+Blazing+-+Kill+(9:17)/Gladrien/standard/statistics`
- Screenshot(s):
image

### CODE REVIEW

File: src/analysis/retail/mage/frost/talents/Cryopathy.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      if (typeof ray.damage !== 'undefined') {
+        const cryopathyStacks = this.selectedCombatant.getBuffStacks(
+          SPELLS.CRYOPATHY_BUFF.id,
+          ray.timestamp,
+        );
+        const cryopathyAmount =
+          ray.damage.reduce((totalAmount, damage) => (totalAmount += damage.amount), 0) *
+          STACK_DMG_INCREASE *
+          cryopathyStacks;
+        cryopathyTotalDamage += cryopathyAmount;
```
Comment (Krealle) [inline src/analysis/retail/mage/frost/talents/Cryopathy.tsx]: Is it intentional you aren't including `damage.absorbed` here?

Comment (earosselot) [reply src/analysis/retail/mage/frost/talents/Cryopathy.tsx]: No, I didn't know that it existed. I'll check on some Tindral log to check the behaviour. Thanks for pointing it out

Comment (earosselot) [reply src/analysis/retail/mage/frost/talents/Cryopathy.tsx]: same log than above, with absorbed 👍 

image

File: src/analysis/retail/mage/frost/talents/Cryopathy.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      }
+    });
+    return (
+      <Statistic size="flexible" category={STATISTIC_CATEGORY.TALENTS}>
+        <BoringSpellValueText spell={TALENTS.CRYOPATHY_TALENT}>
+          <>
+            <p>
+              {formatNumber(cryopathyTotalDamage)} <small>Damage contribution</small>
+            </p>
+            <p>{this.owner.formatItemDamageDone(cryopathyTotalDamage)}</p>
```
Comment (Krealle) [inline src/analysis/retail/mage/frost/talents/Cryopathy.tsx]:
```suggestion
            <ItemDamageDone amount={cryopathyTotalDamage} />
```
You could potentially consider using this component instead to represent the overall dps/% contributed.
Looks like this:
image

**Review by Krealle — APPROVED** [review]
LGTM

File: src/analysis/retail/mage/frost/talents/Cryopathy.tsx (hunk 2 above)
Comment (earosselot) [reply src/analysis/retail/mage/frost/talents/Cryopathy.tsx]: Nice, didn't know it existed. Thanks!

**Pull request merged**